### ISSUE TITLE
give keyboard focus to forms & buttons when user input is required

### ISSUE THREAD

**Issue by mschn** [mschn/wissl#14]
On the login page, keyboard focus should be given to the 'login' form field.
For various dialogs requiring user interaction, the first form item should also have keyboard focus.
For dialogs requiring no interaction, the 'OK' or 'Close' button should have focus.
